src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+:root {
+  --text-color: #3d4e61;
+  --input-box-shadow: 0 1px 8px 0 #3d4e611a;
+  --box-shadow: 0 3px 8px 0 #3d4e6133;
+  --linear-gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+body {
+  /* the big search bar must have top indent 20px */
+  margin-top: 20px;
+}
+
+.form {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.form__bar {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, Helvetica, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's good practice to specify a generic font family, in case none of the preferred fonts are available. Example: `font-family: Avenir, Helvetica, Arial, sans-serif;`.